lm-logs/values.yaml

```diff
@@ -1,5 +1,5 @@
 image:
-  repository: lmmuhammadahsan/lm-logs-k8s-fluentd
+  repository: logicmonitor/lm-logs-k8s-fluentd
   pullPolicy: IfNotPresent
   tag: "latest"
```

Comment:
@ahsan13jan I would suggest to use semantic version tags here. Otherwise "latest" creates problem in future.
If any incompatible change goes, either it stops working or works with other image which is not in sync with helm chart (for ex: sometimes runs new docker image and refers older config map).

Using major versioned image tags would help in long run. For ex: v1 will always to latest image of v1.*.* series.